src/Integration/Laravel/Queue/src/Connector/AsyncAwsSqsConnector.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace AsyncAws\Illuminate\Queue\Connector;
+
+use AsyncAws\Illuminate\Queue\AsyncAwsSqsQueue;
+use AsyncAws\Sqs\SqsClient;
+use Illuminate\Queue\Connectors\ConnectorInterface;
+use Symfony\Component\HttpClient\HttpClient;
+
+class AsyncAwsSqsConnector implements ConnectorInterface
+{
+/**
+     * Establish a queue connection.
+     *
+     * @param  array  $config
+     * @return \Illuminate\Contracts\Queue\Queue
+     */
+    public function connect(array $config)
+    {
+        $clientConfig = [];
+        if ($config['key'] && $config['secret']) {
+            $clientConfig['accessKeyId'] = $config['key'] ?? null;
+            $clientConfig['accessKeySecret'] = $config['secret'] ?? null;
+            $clientConfig['sessionToken'] = $config['token'] ?? null;
+        }
+
+        return new AsyncAwsSqsQueue(
+            new SqsClient($clientConfig, null, HttpClient::create(['timeout'=>60])),
```

Comment:
not need to setup a so high value, Maximum long polling is 20s (https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-short-and-long-polling.html#sqs-long-polling)

Reply:
Thank you